lib/minitest/rails/parallelize.rb

```diff
@@ -25,8 +25,8 @@ module SpecTests # :nodoc:
 module Kernel # :nodoc:
   alias describe_before_minitest_spec_constant_fix describe
   private :describe_before_minitest_spec_constant_fix
-  def describe *args, &block
-    cls = describe_before_minitest_spec_constant_fix(*args, &block)
+  def describe(*, &block)
+    cls = describe_before_minitest_spec_constant_fix(*, &block)
```

Comment:
RuboCop recommends these changes now that we’re on Ruby 3.2+.

Reply:
Not sure this is a good idea because it won't work with JRuby

Reply:
The latest JRuby release is v9.4.9.0, which identifies as Ruby 3.1. So I don’t think you will be able to use Rails 8.0 with JRuby anyway (yet).